### DISCUSSION TITLE
Multiple line chart isn't affected by dashboard filters.

### DISCUSSION THREAD

**Discussion by efsanokur** [apache/superset#26342] [Q&A / Help]
Hey everyone. I have a multiple line chart combining two other charts which don't have the same data source. However, the chart doesn't seem to be affected by any filters set on the filter panel. What might be causing this issue? Or is it how it is supposed to be with multiple line charts? Thanks in advance.

**Comment by sfirke** [comment]
What version of Superset are you using?  What kind of filters?  Especially for charts created before 3.0.0 you might need to make sure there's a variable assigned to the time component of each chart if you are filtering by time.

If they don't have the same data source, filters can still work but the data sources need to have the same variable names for the variables you're filtering.
